css/style.css

```diff
@@ -16,4 +18,293 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+p {
 
+margin-top: 1 rem;
+margin-bottom:0;
+
+
+
+}
+
+ .header {
+
+height: 5rem;
+width: 100%;
+ }
+
+.footer{
+
+    padding: 0 1.5rem;
+    
+    margin: 0 10rem 0 10rem;
+    
+}
+
+
+/* Navigation */
+
+.nav{
+
+display:flex;
+
+
+
+}
+
+
+.logo{
+float:left;
```

Comment:
Please don't use `float` rules in your CSS. Float is the old way of making layouts in CSS and it has now been replaced by Flexbox (which was introduced a few years ago). 
Add flex rules to the parent element to align the logo.